README.md

```diff
@@ -1,17 +1,15 @@
-<div align="center">
-  <a href="https://optimism.io"><img alt="Optimism" src="https://raw.githubusercontent.com/ethereum-optimism/brand-kit/main/assets/svg/OPTIMISM-R.svg" width=320></a>
-  <br />
-  <br />
+<div align="left">
+  <a href="https://lisk.com"><img alt="Lisk" src="https://github.com/LiskHQ/scaffold-lisk/blob/main/packages/nextjs/public/readme-banner.png" width="100%"></a>
 </div>
 
-# 🏗🔴 Scaffold-OP
+# Scaffold-Lisk
 
 <h4 align="center">
   <a href="https://docs.scaffoldeth.io">Documentation</a> |
   <a href="https://scaffoldeth.io">Website</a>
 </h4>
```

Comment:
these are link to the original, since it is mentioned below, we could remove it?